installer/conf/azm-containers-parser.conf

```diff
@@ -0,0 +1,18 @@
+[PARSER]
+    Name         docker
+    Format       json
+    Time_Key     time
+    Time_Format  %Y-%m-%dT%H:%M:%S.%L
+    Time_Keep    On
+    # Command      |  Decoder | Field | Optional Action
+    # =============|==================|=================
+    Decode_Field_As   escaped    log
+
+[PARSER]
+    # http://rubular.com/r/tjUt3Awgg4
+    Name cri
```

Comment:
> should this be renamed to crio?
> since same logging format for all CRI compatiable runtimes so fluentbit guys also renamed cri from cri-o so cri is the right naming.